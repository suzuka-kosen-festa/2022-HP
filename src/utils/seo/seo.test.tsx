import "@testing-library/jest-dom";
import "../../tests/mocks/head.mock";
import { render } from "@testing-library/react";
import { Seo } from ".";

describe("(utils) Seo test", () => {
  test("take snap shot", () => {
    render(
      <Seo
        title={"Re:ROAD | 第57回鈴鹿高専祭"}
        description={"まだ何色にも色づいていない第57回鈴鹿高専祭を今から色づけていきます。"}
      />,
    );
    expect(document.head).toMatchSnapshot();
  });
});
