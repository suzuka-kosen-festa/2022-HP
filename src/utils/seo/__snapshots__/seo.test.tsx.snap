// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(utils) Seo test take snap shot 1`] = `
<head>
  <title>
    Re:ROAD | 第57回鈴鹿高専祭
  </title>
  <meta
    charset="utf8"
  />
  <meta
    content="IE=edge"
    http-equiv="X-UA-Compatible"
  />
  <meta
    content="width=device-width,initial-scale=1"
    name="viewport"
  />
  <meta
    content="まだ何色にも色づいていない第57回鈴鹿高専祭を今から色づけていきます。"
    lang="ja"
    name="description"
  />
  <meta
    content="Re:ROAD | 第57回鈴鹿高専祭"
    name="name"
  />
  <meta
    content="/seo/ogp.png"
    name="image"
  />
  <link
    href="/seo/favicon.ico"
    rel="icon"
  />
  <meta
    content="Re:ROAD | 第57回鈴鹿高専祭"
    property="og:title"
  />
  <meta
    content="まだ何色にも色づいていない第57回鈴鹿高専祭を今から色づけていきます。"
    property="og:description"
  />
  <meta
    content="webpage"
    property="og:type"
  />
  <meta
    content="https://reroad.site/"
    property="og:url"
  />
  <meta
    content="/seo/ogp.png"
    property="og:image"
  />
  <meta
    content="ja_JP"
    property="og:locale"
  />
  <meta
    content="Re:ROAD"
    property="og:site_name"
  />
  <meta
    content="Re:ROAD | 第57回鈴鹿高専祭"
    name="twitter:title"
  />
  <meta
    content="まだ何色にも色づいていない第57回鈴鹿高専祭を今から色づけていきます。"
    name="twitter:description"
  />
  <meta
    content="/seo/ogp.png"
    name="twitter:image"
  />
  <meta
    content="summary_large_image"
    name="twitter:card"
  />
  <meta
    content="@KOSENFESTA"
    name="twitter:site"
  />
  <meta
    content="@KOSENFESTA"
    name="twitter:creator"
  />
  <link
    href="/seo/apple-touch-icon.png"
    rel="apple-touch-icon"
    sizes="180x180"
  />
  <link
    href="/seo/favicon-32x32.png"
    rel="icon"
    sizes="32x32"
    type="image/png"
  />
  <link
    href="/seo/favicon-16x16.png"
    rel="icon"
    sizes="16x16"
    type="image/png"
  />
  <link
    color="#000000"
    href="/seo/safari-pinned-tab.svg"
    rel="mask-icon"
  />
  <meta
    content="#000000"
    name="msapplication-TileColor"
  />
  <meta
    content="#000000"
    name="theme-color"
  />
  <link
    href="/seo/manifest.json"
    rel="manifest"
  />
</head>
`;
